PORTAL/jobs/_utils.py

```diff
@@ -945,8 +945,6 @@ def from_raw(cls, raw, *, name=None):
     def __init__(self, root):
         if not root or root == '.':
             root = CWD
-        else:
-            root = os.path.abspath(os.path.expanduser(root))
```

Comment:
It can still be useful to normalize/absolutize the filename (especially when debugging), even if we don't expand the user.

```python
        elif root.startswith('~'):
            pass
        else:
            root = os.path.abspath(root)
```

We could also disallow relative paths and require that the caller take care of that:

```python
        elif root.startswith('~'):
            pass
        elif not os.path.isabs(root):
            raise ValueError(...)
        else:
            root = os.path.normpath(root)
```

Alternately, that could be indicated by the caller using a keyword argument (where it defaults to resolving the absolute path).